skills/lark-meeting/scripts/meeting_qa_bootstrap.py

```
@@ -0,0 +1,645 @@
+            "",
+            "   ```",
+        ]
+        for line in events_text.strip().splitlines():
+            lines.append(f"   {line}")
+        lines += ["   ```", ""]
+    lines += [
+        "   如果会中共享了文档，可以用以下命令拉取内容：",
+        "   > `--doc-format markdown` 配合 `--format pretty` 输出更短、读起来更快",
+        "",
+        "   ```bash",
+        f"   lark-cli docs +fetch --doc <doc_token / url> --doc-format markdown --format pretty",
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)